app/Models/Room.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Room extends Model
+{
+    use HasFactory;
+
+    protected $casts = [
+        'furniture' => 'array',
+        'services' => 'array',
+        'complimentaries' => 'array'
+    ];
+
+    public function floor()
+    {
+        return $this->belongsTo(Floor::class, 'floor_id');
```

Comment:
Not necessary but just to let you know, we can actually omit the second parameter `floor_id` from the relationship if we follow our Model naming convention.
Example: model name = `Floor` will make migration with table name = `floors`
So, if we are to define a belongsTo() relationship, Laravel will automatically put foreign key = `lower case model name_id` which will result to `floor_id`

In exceptional cases like `manager_id` in `Facility.php`, we must define the second foreign key parameter exclusively. Because we are referring to `User` model's `id` field as `manager_id` in `Facility.php` instead of using automatic `user_id`. Feel free to slack me if you do not understand the concept.